compile-database/compile.py

```diff
@@ -219,7 +259,8 @@ def as_dict(self) -> dict:
             },
             "wip": self.wip,
             "color": self.color,
-            "youtube": self.youtube.as_dict()
+            "youtube": self.youtube.as_dict(),
+            "images": [im.as_dict() for im in self.images]
```

Comment:
Might name the list comprehension iteration variable `img` instead (and rename `ln` to `lnk`), `im` is not obvious for an image